### PR TITLE
V1 meetings - getMeeting.php

### PR DESCRIPTION
**Pull request by fowbi**
### Added

- documentation for getMeeting.php
- addition of description

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -879,6 +879,25 @@ Delete a file
...
+
+### /getMeeting.php [POST /getMeeting.php]
+
+Get a detailed overview of a single meeting.
+
++ Attributes (Meeting)
+
++ Request (multipart/form-data)
+
+    + Body
```
Comment (jenssegers) [inline apiary.apib]: We should define the request using attributes instead of body.

File: apiary.apib [hunk 2]
```diff
@@ -879,6 +879,25 @@ Delete a file
...
 
+## Meetings [/meetings]
+
+### /getMeeting.php [POST /getMeeting.php]
+
+Get a detailed overview of a single meeting.
+
++ Attributes (Meeting)
+
++ Request (multipart/form-data)
```
Comment (jenssegers) [inline apiary.apib]: Decided this should be application/json

File: apiary.apib [hunk 3]
```diff
@@ -1007,3 +1026,31 @@ Delete a file
...
++ duration_minutes: 60 (number)
++ date_timestamp: 554774400 (number)
++ date_day: `01/08/1987` (string)
++ date_hour: `12:00` (string)
++ attending_internal: (array[MeetingUser])
++ attending_external: (array[MeetingUser])
++ creator_id: 1 (number)
++ canceller_id: 1 (number)
++ location: `Street 1, B-1234 City` (string)
++ location_detailed: [type, contact, company] (array[])
```
Comment (jenssegers) [inline apiary.apib]: Is this correct? What is this exactly?

**Comment by Jdww** [comment]
@jenssegers I started from this branch when creating https://github.com/teamleadercrm/api/pull/112 maybe it's better to close this and fix these issues there. Most of them are already fixed.

**Comment by fowbi** [comment]
as @Jdww said, it's better to close this and use #112 with all the fixes already implemented.

**Pull request closed without merging**